subprojects/docs/src/docs/userguide/logging.adoc

```diff
@@ -104,12 +104,7 @@ You can also hook into Gradle's logging system from within other classes used in
 
 === Example: Using SLF4J to write log messages
 
-[source.multi-language-sample,groovy]
-.build.gradle
-----
-include::{samplesPath}/userguide/tutorial/logging/build.gradle[tag=use-slf4j]
-----
-
+include::sample[dir="userguide/tutorial/logging",files="build.gradle[tags=use-slf4j]"]
```

Comment:
it would be good to also update the README to describe the syntax that should be used in the user guide as part of this PR (to make it easier to understand what the proposed syntax is).